scripts/legacy/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from scilpy.io.deprecator import deprecate_script
+from scripts.scil_viz_bundle_screenshot_mni import main as new_main
+
+
+DEPRECATION_MSG = '''
+This script has been renamed scil_bundle_screenshot_mni.py. Please change
```

Comment:
Don't forget to reverify all your sentences "has been renamed X" in your legacy files. (here: scil_viz_bundle_screenshot_mni.py)